### PR TITLE
[manage_secrets] Implement OSP secrets management

### PR DESCRIPTION
**Pull request by rebtoor**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: roles/manage_secrets/molecule/osp_secrets/converge.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
+    cifmw_openshift_login_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"
+    cifmw_manage_secrets_ospsecrets_list:
```
Comment (pablintino) [inline roles/manage_secrets/molecule/osp_secrets/converge.yml:23]: Not sure how are we going to glue this with the rest? Thea idea is to generate this list in the new "edpm_prepare"?

File: roles/manage_secrets/templates/osp_secrets.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% for secret in cifmw_manage_secrets_ospsecrets_list %}
+---
+
+kind: Secret
+apiVersion: v1
+data:
+  {% for username, password in secret.data.items() %}
```
[the remark this answers is not in the document]
Comment (rebtoor) [reply roles/manage_secrets/templates/osp_secrets.yml.j2:7]: yeah, `key`, `value` could be a safer choice here.

File: roles/manage_secrets/molecule/osp_secrets/converge.yml (hunk 1 above)
Comment (rebtoor) [reply roles/manage_secrets/molecule/osp_secrets/converge.yml:23]: yeah, exactly. For now we have only two elements in that list (the `osp-secret` which contains password for every controlplane services and the `octavia-ca-passhprase` which contains, surprise surprise, the passphrase for its CA).
In our next iteration we'll have a secret for each service so we can handle their lifecycle in a more granular way.

**Pull request merged**